.github/workflows/develop-merged.yml

```diff
@@ -10,9 +10,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: ftrip-io/github-ci/build-and-push-to-dockerhub@ef2fd6ac85fe9e0ae63e5979a82a35bd5075bc33
+      - name: Kaniko build
+        uses: aevea/action-kaniko@master
         with:
+          image: ftripio/catalog-service-dev
+          path: ftrip.io.catalog-service
           username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
Should we tag this one as the latest also?

Reply:
I think it tags it as the latest by default.